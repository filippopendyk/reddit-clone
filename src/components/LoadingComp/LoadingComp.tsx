const LoadingComp: React.FC = () => {
    return <h3>Loading posts...</h3>
}

export default LoadingComp;