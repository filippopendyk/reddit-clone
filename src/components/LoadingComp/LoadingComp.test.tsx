import LoadingComp from "./LoadingComp";
import renderer from 'react-test-renderer';

describe('LoadingComp', () => {
    it('Renders without crashing', () => {
        const tree = renderer
        .create(
            <LoadingComp/>
        )
        .toJSON();
        expect(tree).toMatchSnapshot();
    })
})