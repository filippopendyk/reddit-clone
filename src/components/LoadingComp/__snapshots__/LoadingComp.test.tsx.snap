// Vitest Snapshot v1

exports[`LoadingComp > Renders without crashing 1`] = `
<h3>
  Loading posts...
</h3>
`;
